testing/ef-tests/src/macros/sanity_slots.rs

```diff
@@ -1,63 +1,75 @@
 #[macro_export]
 macro_rules! test_sanity_slots {
     ($processing_fn:path) => {
```

Comment:
```suggestion
    () => {
```

if you are no longer using this argument for the macro you can remove it